arangod/GeneralServer/RestHandler.cpp

```diff
@@ -292,6 +454,39 @@ int RestHandler::executeEngine() {
   return TRI_ERROR_INTERNAL;
 }
 
+void RestHandler::generateError(rest::ResponseCode code, int errorNumber,
```

Comment:
At least the function signature looks identical to the one in RestBaseHandler.h.
But the implementation is slightly different.
Can we avoid having yet another generateError function and somehow combine the new one with the existing one?